v2.0/getting-started/kubernetes/installation/integration.md

```diff
@@ -78,7 +78,23 @@ Requires=docker.service
 User=root
 Environment=ETCD_ENDPOINTS=http://<ETCD_IP>:<ETCD_PORT>
 PermissionsStartOnly=true
-ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node -e ETCD_ENDPOINTS= -e HOSTNAME=${HOSTNAME} -e IP= -e NO_DEFAULT_POOLS= -e AS= -e ETCD_AUTHORITY=127.0.0.1:2379 -e ETCD_SCHEME=http -e CALICO_LIBNETWORK_ENABLED=true -e IP6= -e CALICO_NETWORKING_BACKEND=bird -v /var/run/calico:/var/run/calico -v /lib/modules:/lib/modules -v /run/docker/plugins:/run/docker/plugins -v /var/run/docker.sock:/var/run/docker.sock -v /var/log/calico:/var/log/calico calico/node:v1.0.0-beta
+ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node 
+-e ETCD_ENDPOINTS=${ETCD_ENDPOINTS} \
+-e HOSTNAME=${HOSTNAME} \ 
+-e IP= \
+-e NO_DEFAULT_POOLS= \
+-e AS= \
+-e ETCD_AUTHORITY=127.0.0.1:2379 \
+-e ETCD_SCHEME=http \
+-e CALICO_LIBNETWORK_ENABLED=true \
+-e IP6= \
+-e CALICO_NETWORKING_BACKEND=bird \
+-v /var/run/calico:/var/run/calico \
+-v /lib/modules:/lib/modules \
+-v /run/docker/plugins:/run/docker/plugins \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /var/log/calico:/var/log/calico \
```

Comment:
Can we indent the continuing lines by a couple of spaces (makes it easier to read)